### PR TITLE
Allow editing of targets for observation subsets (cloning targets)

### PR DESCRIPTION
**Pull request by toddburnside**
The second commit improves the styling of checkboxes somewhat. It may not be the last word in checkbox styling, but it does allow checks to be seen in both light and dark modes, in both focused and unfocused states.

### CODE REVIEW

File: common/src/main/webapp/less/variables-dark.less [hunk 1]
```diff
@@ -282,6 +282,7 @@
   --checkbox-focus-background: @gray-dark;
   --checkbox-active-background: var(--control-background-color);
   --checkbox-active-focus-background: var(--control-background-color);
+  --checkbox-active-focus-check-color: @selectedTextColor;
```
Comment (cquiroz) [inline common/src/main/webapp/less/variables-dark.less:285]: Would you use a variable here too?

File: explore/src/main/scala/explore/targeteditor/AsterismEditor.scala [hunk 2]
```diff
@@ -154,26 +167,57 @@ object AsterismEditor {
                 val optional =
                   Optional[List[TargetWithId], Target](_.find(_.id === targetId).map(_.target))(
                     target =>
-                      _.map(twid =>
+                      _.map { twid =>
+                        println(s"targetId: $targetId selectedTargetId: ${selectedTargetId.get}")
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targeteditor/AsterismEditor.scala:171]: println?

File: explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala [hunk 3]
```diff
@@ -73,18 +77,51 @@ object SiderealTargetEditor {
 
   implicit val propsReuse: Reusability[Props] = Reusability.derive
 
+  def readonlyView[A](view: View[A]): View[A] = {
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala:80]: Should this go to crystal?

**Review by cquiroz — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala (hunk 3 above)
Comment (toddburnside) [reply explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala:80]: It's possible. I thought we could run it by Raul. There may be a better way to do it.

File: explore/src/main/scala/explore/targeteditor/AsterismEditor.scala (hunk 2 above)
Comment (toddburnside) [reply explore/src/main/scala/explore/targeteditor/AsterismEditor.scala:171]: Yeah. That's a special production ready `println`. 😬 I'll get rid of it.

File: common/src/main/webapp/less/variables-dark.less (hunk 1 above)
Comment (toddburnside) [reply common/src/main/webapp/less/variables-dark.less:285]: I don't know. There is currently a mix of `@` and `var`. I'm not really sure when it's worth creating a new `var` from an `@`.

**Pull request merged**